kernel-modules/build/build-kos

```diff
@@ -85,13 +95,11 @@ build_ko() (
                 touch "${MODULE_BASE_DIR}/${module_version}/.collector-ebpf-${kernel_version}.unavail"
                 return 0
             fi
-        fi
 
-        echo "Building collector eBPF probe for kernel version ${kernel_version} and module version ${module_version}."
-
-        KERNELDIR="$kernel_build_dir" BUILD_ROOT="${KERNEL_SRC_DIR}" make -j 6 -C bpf
-
-        collector_probe="bpf/probe.o"
+            echo "Building collector eBPF probe for kernel version ${kernel_version} and module version ${module_version}."
+            KERNELDIR="$kernel_build_dir" BUILD_ROOT="${KERNEL_SRC_DIR}" FALCO_DIR="$(pwd)/bpf" make -j 6 -C /probe
+            collector_probe="/probe/probe.o"
```

Comment:
[nit] We could eliminate duplicated code if we had a variable `probe_directory` set by default to `/probe` and changed to `bpf` when compiling RHEL 7 kernels, then the compilation command could be:
```make
KERNELDIR="$kernel_build_dir" BUILD_ROOT="${KERNEL_SRC_DIR}" FALCO_DIR="$(pwd)/bpf" make -j 6 -C "${probe_directory}"
collector_probe="${probe_directory}/probe.o"
```

The `FALCO_DIR` variable would be ignored by the catch-all probe and we could go back to having a single command for all eBPF probes, WDYT?

Reply:
That seems a lot neater! It also looks like RHEL-7 is using the new probe at the moment, which is odd. I'll look into it when making these changes.